examples/aws_flows/main.tf

```diff
@@ -0,0 +1,14 @@
+provider "aws" {
+  region = "us-west-1"
+}
+
+provider "illumio-cloudsecure" {
+  client_id     = var.illumio_cloudsecure_client_id
+  client_secret = var.illumio_cloudsecure_client_secret
+}
+
+module "aws_flows_dev" {
+  source        = "github.com/illumio/terraform-illumio-cloudsecure//modules/aws_flows?ref=v1.2.0"
+  role_arn      = var.illumio_cloudsecure_role_arn
```

Comment:
Modify this example to add an `aws_account` to show how those should be used together, instead of taking the role ARN as a variable.